podium/talks/models.py

```diff
@@ -20,11 +20,20 @@ class Talk(models.Model):
         max_length=1, choices=TALK_STATUS_CHOICES,
         default='S')
 
+    def get_absolute_url(self):
+        return f'/talks/talks/{self.id}/'
+
 
 class Session(models.Model):
     date = models.DateField()
     description = models.TextField(
         blank=True, help_text='Any special theme or info about the session.')
 
     def approved_talks(self):
-        return self.talks_available.filter(status='A')
+        sets = [
```

Comment:
Rather than doing two separate queries by status, and doing a union, try taking advantage of Django's built-in filter field lookups:  https://docs.djangoproject.com/en/1.11/ref/models/querysets/#id4
For example, this might be a good place to use the "in" lookup, like `.filter(status__in=('A', 'C'))`